modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -26,65 +26,63 @@ def show
 
         def appoint_organization
           validate_request!(ClaimsApi::V2::ParamsValidation::PowerOfAttorney)
-          poa_code = parse_and_validate_poa_code
-          unless poa_code_in_organization?(poa_code)
+          validated_poa_code = shared_form_validation('2122')
+          unless poa_code_in_organization?(validated_poa_code)
             raise ::Common::Exceptions::UnprocessableEntity.new(detail: 'POA Code must belong to an organization.')
           end
 
-          submit_power_of_attorney(poa_code)
+          submit_power_of_attorney(validated_poa_code, '2122')
         end
 
-        def appoint_individual
+        def submit2122a
+          validated_poa_code = shared_form_validation('2122a')
           validate_request!(ClaimsApi::V2::ParamsValidation::PowerOfAttorney)
-          poa_code = parse_and_validate_poa_code
-          if poa_code_in_organization?(poa_code)
+          if poa_code_in_organization?(validated_poa_code)
```

Comment:
I'm curious if we want this. The 2122a form does allow for two additional types of individuals in 15-b other than attorney and agent. I vaguely remember some research that these other options were rarely used, caused confusion, and they might be removed in a future form version.

I wrote the schema to accept them, but this line would reject them.